### ISSUE TITLE
Endless Mini full recovery from SD card

### ISSUE THREAD

**Issue by dumblob** [dumblob/tmp#2]
Hi @dsd,

we've been in contact some time ago (I'm the author of https://community.endlessos.com/t/endless-mini-new-installation-missing-arm-image/11430/17 ) and after those months of my Endless Mini just lying around in a non-functional state, I decided to **give up warranty** and reflash the device.

Could you please briefly describe how do I fully reflash the Endless Mini? I found https://github.com/endlessm/u-boot-meson/commits/master and https://github.com/endlessm/linux-meson but that's too low level (I don't want to compile so much stuff).

I imagine steps like [Arch Linux Arm ODROID-C1 installation](https://archlinuxarm.org/platforms/armv7/amlogic/odroid-c1#installation ) (Odroid C1 uses the same SoC, but a different "dtb" HW description file which I tried to flash to the SD card present in Endless Mini, but it didn't boot).

Basically I don't care which linux distribution I'll have on the Endless Mini. I just need a small wifi-capable Linux machine with a working usb and ethernet connections.

**Comment by dsd** [comment]
I sent you a message on the forum with instructions.
